examples/quantization_aware_training/torch/anomalib/main.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import subprocess
+from copy import deepcopy
+from pathlib import Path
+from typing import List
+
+import torch
+from anomalib import TaskType
+from anomalib.data import MVTec
+from anomalib.deploy import ExportType
+from anomalib.engine import Engine
+from anomalib.models import Stfpm
+
+import nncf
+
+HOME_PATH = Path.home()
+DATASET_PATH = HOME_PATH / ".cache/nncf/datasets/mvtec"
+CHECKPOINT_PATH = HOME_PATH / ".cache/nncf/models/stfpm_mvtec"
+ROOT = Path(__file__).parent.resolve()
+FP32_RESULTS_ROOT = ROOT / "fp32"
+INT8_RESULTS_ROOT = ROOT / "int8"
+CHECKPOINT_URL = "https://huggingface.co/alexsu52/stfpm_mvtec_capsule/resolve/main/qat/model.ckpt"
```

Comment:
we found this is not a good practice to share stuff from a private account. We should revise the samples where there is such a behavior. The best alternative is to use OpenVINO S3.

Reply:
done